data/sql/updates/pending_db_world/rev_1681594798691444614.sql

```diff
@@ -0,0 +1,7 @@
+--
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 16473;
+
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 16473) AND (`source_type` = 0) AND (`id` IN (3));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(16473, 0, 3, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 11, 29680, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Spectral Performer - On Death - Cast Curtain Call');
+
```

Comment:
You can remove the UPDATE query if it already has an AIName
```suggestion
--
DELETE FROM `smart_scripts` WHERE (`entryorguid` = 16473) AND (`source_type` = 0) AND (`id` IN (3));
INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
(16473, 0, 3, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 11, 29680, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Spectral Performer - On Death - Cast Curtain Call');
```

Reply:
I use Keira and don't _really_ know SQL I don't know why it's there ~